tests/source/issue-3343/two.rs

```diff
@@ -0,0 +1,14 @@
+// rustfmt-version: Two
+// rustfmt-config: inline_attribute_width.toml
+
+#[cfg(feature = "alloc")]
+use core::slice;
+
+#[cfg(feature = "alloc")]
+use total_len_is::_50__;
+
+#[cfg(feature = "alloc")]
+use total_len_is::_51___;
+
+/// comment
```

Comment:
Can you write in the comment that this is to test `is_sugared_doc` property of an attribute ?